source/fundamentals/serialization/guid-serialization.txt

```diff
@@ -24,87 +24,104 @@ In this guide, you can learn how to serialize **globally unique identifiers**
 (`GUIDs <https://learn.microsoft.com/en-us/dynamicsax-2012/developer/guids>`__),
 also known as **universally unique identifiers** (UUIDs).
 
-.. tip::
+A Short History of MongoDB GUIDs
+--------------------------------
 
-   In MongoDB applications, ``ObjectId`` can be used as a unique identifier for
-   a document. Consider using ``ObjectId`` in place of a GUID with MongoDB
-   applications where possible.
+A GUID is a 16-byte integer that you can use as a unique ID for a MongoDB document.
+The following code block shows an example GUID:
 
-A GUID is a 16-byte integer that you can use as a unique ID for a MongoDB document. 
-Originally, GUIDs in MongoDB were represented as ``BsonBinaryData`` values of subtype 3. 
-Subtype 3 did not standardize the byte order during serialization, which led to 
-inconsistent serialization across MongoDB drivers. 
-To standardize the byte order and ensure consistent serialization across drivers, we 
-created ``BsonBinaryData`` subtype 4.
+.. code-block::
 
-.. note::
-   
-   Use ``BsonBinaryData`` subtype 4 for all new GUIDs. 
+  00112233-4455-6677-8899-aabbccddeeff
 
-GuidRepresentationMode
-----------------------
+Originally, MongoDB represented GUIDs as ``BsonBinaryData``
+values of subtype 3. Because subtype 3 didn't standardize the byte order of GUIDs
+during encoding, different MongoDB drivers encoded GUIDs with different byte orders.
```

Comment:
Q: is there a time frame or C# release # that might be useful to reference here instead of broadly saying "originally"?

Reply:
i don't know, but i can mark it as a question for the tech reviewer